src/test/kotlin/com/jayrave/moshi/pristineModels/FieldMappingImplTest.kt

```diff
@@ -219,7 +219,7 @@ class FieldMappingImplTest {
         mapping.clearLastReadValueInCurrentThread()
         assertThat(mapping.lastReadValueInCurrentThread()).isNull()
     }
-
+*/
```

Comment:
In all the commented out tests, I think we can replace the assertion on `lastReadValueInCurrentThread` to be directly on `fieldMapping.read..` since that is supposed to return the value as is now